libs/gltfio/src/AssetLoader.cpp

```diff
@@ -100,11 +100,19 @@ uint32_t computeBindingOffset(const cgltf_accessor* accessor) {
     return uint32_t(accessor->offset + accessor->buffer_view->offset);
 }
 
+const char* getNodeName(const cgltf_node* node, const char* defaultNodeName) {
+  if (node->name) return node->name;
+  if (node->mesh && node->mesh->name) return node->mesh->name;
+  if (node->light && node->light->name) return node->light->name;
+  return defaultNodeName;
+}
+
 struct FAssetLoader : public AssetLoader {
     FAssetLoader(const AssetConfiguration& config) :
             mEntityManager(config.entities ? *config.entities : EntityManager::get()),
             mRenderableManager(config.engine->getRenderableManager()),
             mNameManager(config.names),
+	    mDefaultNodeName(config.defaultNodeName),
```

Comment:
```suggestion
        mDefaultNodeName(config.defaultNodeName),
```

Reply:
This initialization order will cause warnings with some compilers, this should be the last entry in the initializer list because it occurs later in the sequence of class member declarations.  Also, no tabs.  :)